connectors/randori/connector/full_import.py

```diff
@@ -22,11 +24,21 @@ def import_collection(self):
         """
         context().logger.debug('Import collection started')
 
-        ## this need to be implemeted as we build data handler
-        # collections = context().asset_server.get_data_source()
-        #
-        # for node in ['cluster', 'node', 'application', 'container', 'account', 'user']:
-        #     getattr(self.data_handler, 'handle_' + node.lower())(collections)
+        # TODO
+        query = "eyJjb25kaXRpb24iOiJBTkQiLCJydWxlcyI6W3siY29uZGl0aW9uIjoiT1IiLCJydWxlcyI6W3sibGFiZWwiOiJtZWRpdW0iLCJjb25kaXRpb24iOiJBTkQiLCJydWxlcyI6W3siaWQiOiJ0YWJsZS5jb25maWRlbmNlIiwiZmllbGQiOiJ0YWJsZS5jb25maWRlbmNlIiwidHlwZSI6ImludGVnZXIiLCJpbnB1dCI6Im51bWJlciIsIm9wZXJhdG9yIjoiZ3JlYXRlcl9vcl9lcXVhbCIsInZhbHVlIjo2MH0seyJpZCI6InRhYmxlLmNvbmZpZGVuY2UiLCJmaWVsZCI6InRhYmxlLmNvbmZpZGVuY2UiLCJ0eXBlIjoiaW50ZWdlciIsImlucHV0IjoibnVtYmVyIiwib3BlcmF0b3IiOiJsZXNzX29yX2VxdWFsIiwidmFsdWUiOjc0fV19LHsibGFiZWwiOiJoaWdoIiwiY29uZGl0aW9uIjoiQU5EIiwicnVsZXMiOlt7ImlkIjoidGFibGUuY29uZmlkZW5jZSIsImZpZWxkIjoidGFibGUuY29uZmlkZW5jZSIsInR5cGUiOiJpbnRlZ2VyIiwiaW5wdXQiOiJudW1iZXIiLCJvcGVyYXRvciI6ImdyZWF0ZXJfb3JfZXF1YWwiLCJ2YWx1ZSI6NzV9LHsiaWQiOiJ0YWJsZS5jb25maWRlbmNlIiwiZmllbGQiOiJ0YWJsZS5jb25maWRlbmNlIiwidHlwZSI6ImludGVnZXIiLCJpbnB1dCI6Im51bWJlciIsIm9wZXJhdG9yIjoibGVzc19vcl9lcXVhbCIsInZhbHVlIjoxMDB9XX1dLCJ1aV9pZCI6ImNvbmZpZGVuY2UifSx7InVpX2lkIjoidW5hZmZpbGlhdGVkIiwiY29uZGl0aW9uIjoiT1IiLCJydWxlcyI6W3sidWlfaWQiOiJzaG93X25vX2FmZmlsaWF0aW9uIiwiaWQiOiJ0YWJsZS5hZmZpbGlhdGlvbl9zdGF0ZSIsImZpZWxkIjoidGFibGUuYWZmaWxpYXRpb25fc3RhdGUiLCJ0eXBlIjoib2JqZWN0IiwiaW5wdXQiOiJ0ZXh0Iiwib3BlcmF0b3IiOiJlcXVhbCIsInZhbHVlIjoiTm9uZSJ9XX1dLCJ1aV9pZCI6ImJhc2UifQ=="
```

Comment:
for the code readability I would prefer to see the query as a normal json or string (`'{"condition":"AND","rules":[{"condition":"OR".....'`) and then do `query = b64encode(query.encode())` as it is there below commented out. 
Having it as bease64 right in the code hardcoded reduces comprehension and is less maintainable.

Reply:
That's that end goal. Before we merge it develop, that will be implemented. For now there are some questions that needs to be answered, I'm waiting for that.